backend/db/migrations/20200301163928-migrate-entry-data.js

```diff
@@ -1,5 +1,5 @@
 import OldEntry from '../../models/oldEntry'
-import Entry from '../../models/Entry'
+import Entry from '../../models/entry'
```

Comment:
@JohnPMurray I had to do this to get the migrations to run. Did this pop up when you were working on it?

Reply:
not sure it worked for me, but if this fixed an issue for you thats fine